src/Components/login/materialStyles.js

```diff
@@ -0,0 +1,23 @@
+const messageSignInStyle = {
+  text: "Enter your detalls below.",
+  messageColor: { color: "rgb(150, 157, 166)" },
+  borderBottonStyle: { borderBottom: "1px solid rgb(179, 175, 177)" },
+};
+
+const failedLoginMessageStyle = {
+  text: "Oops! That email or password combination is not valid.",
+  messageColor: { color: "rgb(237, 85, 151)" },
```

Comment:
create a variable (inside this file) called `messageColor` and re use it in here and on line 15